packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/index.ts

```diff
@@ -27,15 +24,14 @@ export const render = () => {
             <mj-text css-class="primary-text">It takes two to sync. Syncing another device with Firefox privately keeps your bookmarks, passwords and other Firefox data the same everywhere you use Firefox.</mj-text>
           </mj-column>
         </mj-section>
-        </mj-section>
         ${button}
         ${appBadges}
         <mj-section>
           <mj-column>
-            <mj-text css-class="secondary-text">This is an automated email; if you received it in error, no action is required. For more information, please visit Mozilla Support.</mj-text>
-            <mj-text css-class="secondary-text"><span>Mozilla. 2 Harrison St, #175, San Francisco, CA 94105</span></mj-text>
-            <mj-text css-class="secondary-text"><span>Mozilla Privacy Policy</span></mj-text>
-            <mj-text css-class="secondary-text"><span>Firefox Cloud Terms of Service</span></mj-text>
+            <mj-text css-class="secondary-text mt-6 mb-3">This is an automated email; if you received it in error, no action is required.For more information, please visit <span>Mozilla Support<span></mj-text>
```

Comment:
I think `mt-6` isn't being applied, at least when I'm running it locally. Maybe it also needs to be `mt-6 div`?